application/views/survey_responses_view.py

```diff
@@ -43,14 +43,17 @@ def add_survey_response(case_id):
         if not file_name:
             return make_response(MISSING_DATA, 404)
 
-        upload_success = survey_response.add_survey_response(case_id, file, file_name, survey_ref)
-
-        if upload_success == "False":
+        try:
+            log.info('Processing file')
+            survey_response.add_survey_response(case_id, file, file_name, survey_ref)
+        except FileTooSmallError:
+            log.error('File too small')
```

Comment:
You're going to end up with two log statements exactly the same here, as you're logging this above. This could probably just be removed.